<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1526</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 246, 232)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="GetInstrumentID">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>630</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>get instrument ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenDevice">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>670</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZCheck_Off">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>650</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>zch off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ZCheck_On">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>590</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>zch on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IncrementVoltage">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>620</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecrementVoltage">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>680</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetVoltage">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>740</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>getvoltage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IV_Cycle">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cycle?</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="IV_PlotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="CurrentSample_PlotWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>621</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentSample_Start">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>760</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentSample_Pause">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>760</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentSample_Stop">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>760</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_Start">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_Pause">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_Stop">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_SavePath_Field">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_StartVoltage_Field">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_EndVoltage_Field">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_VoltageIncrement_Field">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_IncreaseSweepCount">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>120</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_DecreaseSweepCount">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>140</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_SweepControls_Label">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep Controls</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_SweepsCount_Label">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep count</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_StartVoltage_Label">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_EndVoltage_Label">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End Voltage</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_VoltageIncrement_Label">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage Increment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_AddToSweepQueue">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>450</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to sweep queue</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_Pause_Label">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_Pause_Field">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="IV_SweepsCount_Field">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>120</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_LoadParameters">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>370</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#IV_LoadParameters
{
	font: 75 8pt &quot;Arial&quot;;
	rgb: (15, 159, 180);
}
</string>
    </property>
    <property name="text">
     <string>Load Parameters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_SaveParameters">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>370</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="IV_Progress_Label">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_SaveSelectedSweeps">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_SaveAllSweeps">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save All Sweeps</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="IV_SweepTable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>521</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_SetSavePath">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Path</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>1511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>290</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="IV_SweepAverageTable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>170</y>
      <width>521</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="VoltageOff">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>770</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage Off</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VoltageOn">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>710</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage On</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IV_Autoscale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoscale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_ZoomInX">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_ZoomInY">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_ZoomOutX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="IV_ZoomOutY">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-y</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CurrentSample_Autoscale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Autoscale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentSample_ZoomOutY">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CurrentSample_ZoomInY">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>+y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1526</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
